mc_all.sas

```diff
@@ -2082,7 +2082,7 @@ data _null_;
 /* remove all formats for write purposes - some have long underlying decimals */
   format _numeric_ best30.29;
   length __dataline $32767;
-  __dataline=catq('cqsm',&datalines);
+  __dataline=catq('cqtmb',&datalines);
```

Comment:
so we are left aligning the numerics, and otherwise keeping the leading *b*lanks.  Looks great!